### DISCUSSION TITLE
Skylar周刊第28期 - blog

### DISCUSSION THREAD

**Discussion by giscus** [0522skylar/weekly#2] [General]
# Skylar周刊第28期 - blog

端午节，我终于在下午出去走了走，但是风景不如林间小道，想念公司附近的小路，散步走起来很舒服。端午节和高考一起放假，高考前三天后三天，再加上高考三天，弟弟的高中放9天假期，这样的假期全家一起去旅游才好。

http://localhost:4321/posts/28-blog

**Comment by skylar-sun** [comment]
😀
